guides/common/modules/proc_configuring-logging-to-journal-or-file-based-logging.adoc

```diff
@@ -0,0 +1,42 @@
+[id="Configuring_Logging_to_Journal_or_File-based_Logging_{context}"]
+= Configuring Logging to Journal or File-based Logging
+
+{Project} uses file-based logging by default.
+You can use the `{foreman-installer}` command to reconfigure logging.
+
+.Procedure for configuring logging with Journal
+* Enter the following `{foreman-installer}` command to configure logging to the `journald` service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-logging-layout pattern \
+--foreman-logging-level info \
+--foreman-logging-type journald \
+--foreman-proxy-log JOURNAL
+----
+
+Journal now forwards log messages to `rsyslog`, and `rsyslog` writes the log messages to `/var/log/messages`.
+After this change, the log messages do not appear in `/var/log/foreman/production.log` or `/var/log/foreman-proxy.log`.
+
+.Procedure for configuring file-based logging
+* Enter the following `{foreman-installer}` command to configure the default file-based logging:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--reset-foreman-logging-layout \
+--reset-foreman-logging-level \
```

Comment:
Similar as above, I'd also leave the logging level parameter out.

Reply:
Removing the line.